lib/sqlalchemy/sql/selectable.py

```diff
@@ -3477,7 +3482,7 @@ def set_label_style(self, style: SelectLabelStyle) -> Self:
         "first in order to create "
         "a subquery, which then can be selected.",
     )
-    def select(self, *arg: Any, **kw: Any) -> Select[Any]:
+    def select(self, *arg: Any, **kw: Any) -> Select[Unpack[Tuple[Any, ...]]]:
```

Comment:
**Michael Bayer** (zzzeek) wrote:

TupleAny

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986

Reply:
Done